package.json

```diff
@@ -5,13 +5,12 @@
   "dependencies": {
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-scripts": "3.0.1",
-    "sinon": "^7.4.1"
+    "react-scripts": "3.0.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=3000 react-scripts start",
```

Comment:
Has this changed to allow for consistency when Travis runs?

Reply:
yes